usr/bin/nxdk-pkg-config

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
+export PKG_CONFIG_LIBDIR=""
+export PKG_CONFIG_LIBDIR="${PKG_CONFIG_LIBDIR}:${NXDK_DIR}/usr/lib/pkgconfig"
```

Comment:
why not:
```sh
export PKG_CONFIG_LIBDIR="${NXDK_DIR}/usr/lib/pkgconfig"
export PKG_CONFIG_LIBDIR="${PKG_CONFIG_LIBDIR}:${NXDK_DIR}/usr/local/lib/pkgconfig"
```

Reply:
Because it obscures the fact that `PKG_CONFIG_LIBDIR` should be empty, which is very critical, because otherwise packages on the host system might be included.

These 3 lines should probably be split into 2 blocks:
1. Clearing, to ensure the host is not accessed
2. Appending new directories for nxdk library paths

I should add comments to each block to clarify what's going on